docs/Developers/Useful_Links.rst

```diff
@@ -37,7 +37,7 @@ rST-Links
 coala-Links
 -----------
 
--   `coala Shortlinks <coala.io/links>`_
+-   `coala Shortlinks <./Shortlinks.html>`_
```

Comment:
Shouldn't this be a direct link to the Shortlinks file, just as the link to ```coala Newcomers' Guide``` 5 lines down?

Reply:
This is a direct link to the shortlinks file, you may check the same https://deploy-preview-5915--coala-api.netlify.com .

Reply:
So it's just for the preview right? Once the PR is ready to be merged that would be changed to a direct link?

Reply:
No, this will create problems if coala changes its domain.

Reply:
@ayan-b, won't that be true for the other links as well?

Reply:
Yes, but that is unrelated to the corresponding issue, so that should be a separate issue and separate PR. :)

Reply:
If other reviewers agree, this can create a lot of newcomer's issues :smile:.